messages/TypescriptEslintStrategy.js

```diff
@@ -2,6 +2,5 @@ module.exports = {
 	"FileNotIncludedByTsConfig": "'%s' does not reside in a location that is included by your %s 'include' attribute.",
 	"MissingTsConfigFromCwd": "Unable to find '%s' in current directory '%s'. Please change to a directory that contains '%s' or specify it using the --tsconfig flag",
 	"InvalidNameTsConfigFromOptions": "File '%s' specified by the --tsconfig flag must be named '%s'",
-	"NotAFileTsConfigFromOptions": "Unable to find '%s' at location '%s' specified by the 'tsconfig' option",
-	"MultipleTsConfigs": "'%s' was found in current directory at location '%s', '%s' was also specified by the --tsconfig flag. Please change to another directory or remove the --tsconfig flag."
+	"NotAFileTsConfigFromOptions": "Unable to find '%s' at location '%s' specified by the 'tsconfig' option"
```

Comment:
I checked the the docs/examples for the run command, they don't need to be changed based on this change.